examples/README.md

```diff
@@ -0,0 +1,41 @@
+ # Module atracker
```

Comment:
Create a directory under examples folder and copy the respective files (main.tf, versions.tf, variables.tf, README etc) inside that

Reply:
11c13c2ff9964f8f18aae14c2db02f7c8b9051fd